flink-end-to-end-tests/test-scripts/test_shaded_presto_s3.sh

```diff
@@ -22,15 +22,11 @@
 source "$(dirname "$0")"/common.sh
 source "$(dirname "$0")"/common_s3.sh
 
-s3_put $TEST_INFRA_DIR/test-data/words $ARTIFACTS_AWS_BUCKET flink-end-to-end-test-shaded-presto-s3
```

Comment:
Could we still do the put (to guard against accidental deletes etc.) without affecting test stability?

Reply:
I think this is good point. We can just remove trap with deletion of the file atm.